scss/_variables.scss

```diff
@@ -79,6 +79,7 @@ $spacers: (
   )
 ) !default;
 $border-width:               1px !default;
+$input-border-width:         .0625rem !default;
```

Comment:
Use the existing (well, it's in the latest `v4-dev`) `$input-btn-border-width` variable instead.